### PR TITLE
[feat] Navigation doc site

### PR DESCRIPTION
**Pull request by Hamdrive**
Created navigation site, contains navigation for Ecommerce website (Desktop version)

image

### CODE REVIEW

File: components/navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  top: 0;
+  left: 0;
+  padding: var(--size-9) 0;
+}
+
+.nav-main {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1200px;
```
Comment (shivampandey0) [inline components/navigation/navigation.css:16]: why not 100% or 100vw? 
or
80 or 90 % or vw

**Review by FarhanMobashir — COMMENTED** [review]
all good ✔️

File: components/navigation/navigation.css [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  flex-basis: 45%;
+  position: relative;
+}
+
+.search-icon {
+  font-size: var(--size-11);
+  margin: auto 0;
+  position: absolute;
+  top: 25%;
+  left: 3%;
```
Comment (shivampandey0) [inline components/navigation/navigation.css]: doesn't it look like being too specific?

**Review by shivampandey0 — COMMENTED** [review]
Looks good, asked some questions.

File: components/navigation/navigation.css (hunk 2 above)
Comment (Hamdrive) [reply components/navigation/navigation.css]: Correct!
Have recitified this with proper positioning, thank you for pointing it out.

**Comment by Hamdrive** [comment]
Refactored changes based on reviews given, merging to dev

**Pull request merged**